api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -49,4 +49,18 @@
 		</createTable>
 	</changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="order_set" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add a category field to the order_Set table
```

Comment:
The table name in the above description is not spelled correctly.

Reply:
I think I have rectified that.